### PR TITLE
Modified the peer ops file

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Added the return types in function.
Removed the non-matching docstrings.

Fixes: #182
Updates: #172

Signed-off-by: aujjwal-redhat <aujjwal@redhat.com>

### CODE REVIEW

File: support/ops/gluster_ops/peer_ops.py [hunk 1]
```diff
@@ -44,7 +31,7 @@ def peer_probe(self, server: str, node: str):
             raise Exception(ret['msg']['opErrstr'])
 
         self.logger.info(f"Successfully ran {cmd} on {node}")
-        return ret
```
Comment (srijan-sivakumar) [inline support/ops/gluster_ops/peer_ops.py:47]: Any specific reason as to why the return value is being removed?
What if the TC demands that the response of peer probe is to be checked if it remains the same after multiple peer probe and detach commands. There are certain glusto TCs which perform this type of operation.

File: support/ops/gluster_ops/peer_ops.py [hunk 2]
```diff
@@ -83,27 +57,13 @@ def peer_detach(self, node: str, server: str, force: bool = False):
...
```
Comment (srijan-sivakumar) [inline support/ops/gluster_ops/peer_ops.py:86]: Same as above.

File: support/ops/gluster_ops/peer_ops.py [hunk 3]
```diff
@@ -117,9 +77,9 @@ def peer_status(self, node: str):
...
```
Comment (srijan-sivakumar) [inline support/ops/gluster_ops/peer_ops.py:120]: Same as above.

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]

File: support/ops/gluster_ops/peer_ops.py (hunk 1 above)
Comment (aujjwal-redhat) [reply support/ops/gluster_ops/peer_ops.py:47]: Oh ok. Will add ret then.

File: support/ops/gluster_ops/peer_ops.py (hunk 2 above)
Comment (aujjwal-redhat) [reply support/ops/gluster_ops/peer_ops.py:86]: Ack

**Review by neesingh-rh — APPROVED** [review]

File: support/ops/gluster_ops/peer_ops.py (hunk 1 above)
Comment (aujjwal-redhat) [reply support/ops/gluster_ops/peer_ops.py:47]: @srijan-sivakumar is it fine now?

**Review by srijan-sivakumar — APPROVED** [review]
LGTM

**Pull request merged**